app/partials/app.cjsx

```diff
@@ -72,7 +70,7 @@ module.exports = React.createClass
   getEnv: ->
     reg = /\W?env=(\w+)/
     browser_env = window?.location?.search?.match(reg)
-    @state?.env || browser_env || 'staging'
+    @state?.env || browser_env[1] || 'staging'
```

Comment:
Wait, I'm an idiot. The API client config will have the environment already, so we can just read it from there instead of getting it from `window.location` again.